python/paddle/fluid/io.py

```diff
@@ -473,79 +478,154 @@ def save_checkpoint(executor,
 
     :param executor
     :param checkpoint_dir
-    :param max_num_checkpoints
-    :param save_interval_secs
+    :param trainer_id
+    :param is_chief
     :param main_program
+    :param max_num_checkpoints
     """
     if checkpoint_dir is None:
-        checkpoint_dir = os.getcwd()
+        raise ValueError("The values of 'checkpoint_dir' should not be None")
```

Comment:
`checkpoint_dir` should not be None.

Reply:
```checkpoint_dir``` should be checked in ```Trainer.py```, A property directory will be given by  ```Trainer.py```.